### PR TITLE
[52]:... Test for log message struct

### PR DESCRIPTION
**Pull request by Darrellbor**
PR for #52 
Implemented `TestLog_Message` and check messages for 
- [x] Module 
- [x] Line
- [x] File
- [x] Time 
- [x] Pid 
- [x] Tid 
- [x] Text

### CODE REVIEW

**Comment by Darrellbor** [comment]
@gavv PR ready for review

**Comment by gavv** [comment]
> @gavv PR ready for review

Thanks for PR! You can cast @roc-streaming/go instead of me because there are others who can review PRs too.

**Comment by Darrellbor** [comment]
oh ok 
noted

**Comment by Darrellbor** [comment]
@roc-streaming/go ready for re-review

File: roc/log_test.go [hunk 1]
```diff
@@ -91,6 +93,80 @@ func TestLog_Func(t *testing.T) {
...
+	defer close(tw.ch)
+
+	SetLoggerFunc(func(msg LogMessage) {
+		if msg.Level == LogTrace {
+			msgLevel := strconv.Itoa(int(msg.Level))
+			msgLine := strconv.Itoa(msg.Line)
+			msgPid := strconv.Itoa(int(msg.Pid))
+			msgTid := strconv.Itoa(int(msg.Tid))
+			msgTime := strconv.Itoa(int(msg.Time))
+			byteMsg := msgLevel + ":" +
```
Comment (ortex) [inline roc/log_test.go:110]: I don't think we need to convert msg to string and then from string to msg in this test. I guess we can just use `ch := make(chan LogMessage, 1)` here

File: roc/log_test.go [hunk 2]
```diff
@@ -91,6 +95,47 @@ func TestLog_Func(t *testing.T) {
...
+
+	ctx, _ := OpenContext(ContextConfig{})
+
+	select {
+	case entry := <-tw.ch:
+		var msg LogMessage
+		err := json.Unmarshal([]byte(entry), &msg)
+		require.NoError(t, err)
+
+		require.Equal(t, LogTrace, LogLevel(msg.Level), "Expected log level to be trace")
```
Comment (ortex) [inline roc/log_test.go:123]: no need to wrap `LogLevel(msg.Level)`. msg.Level already LogLevel

**Review by ortex — APPROVED** [review]

**Comment by gavv** [comment]
Thanks!

**Pull request merged**